packages/vscode-extension/src/webview/components/PreviewLoader.tsx

```diff
@@ -20,20 +20,52 @@ const startupStageWeightSum = StartupStageWeight.map((item) => item.weight).redu
   0
 );
 
+const BREAKPOINT_SMALL_ROTATED = 450;
+const BREAKPOINT_MEDIUM_ROTATED = 700;
```

Comment:
NIT:
```suggestion
const BREAKPOINT_SMALL_LANDSCAPE = 450;
const BREAKPOINT_MEDIUM_LANDSCAPE = 700;
```
makes more sense to me to have PORTRAIT-LANDSCAPE dichotomy, instead of `PORTRAIT`-`ROTATED`.

Reply:
Renamed in 47baec2c9aa71728ff1a2d92d76396052a2dae55